flowplayer.hlsjs.js

```diff
@@ -616,6 +616,6 @@
         });
     }
 
-}.apply(null, (typeof module === 'object' && module.exports)
-    ? [require('flowplayer'), require('hls.js')]
-    : [window.flowplayer, window.Hls]));
+};
+if (typeof module === 'object' && module.exports) module.exports = extension.bind(undefined, require('hls.js'));
+else if (window.flowplayer) extension(window.flowplayer, window.Hls);
```

Comment:
@nnarhinen: order of arguments is reversed, should be `extension(window.Hls, window.flowplayer)` ;-)